sites/postcss-preset-env/blog/can-use-boolean-custom-media.md

```diff
@@ -0,0 +1,35 @@
+---
+title: You can now use true and false in custom media queries!
+description: This allows you to easily debug a condition without a major refactor.
+date: 2023-01-25
+---
+
+Since the release of PostCSS Custom Media version 9, you can now use `true` and `false` in your custom media queries. This allows you to easily debug a condition without a major refactor.
```

Comment:
We should make `PostCSS Custom Media version 9` a link.

Is it better to link to the GitHub repo or to npm?
Maybe best if we have some consistency going forward :)

Reply:
That's right! I had the same doubt and missed this. I'm actually not sure, I myself prefer GH all the time because if I have any doubt I will use the search to search for any code and / or dive into the source.

Thoughts?

Reply:
I have added GH but happy to change

Reply:
GH is perfect!